src/build_terminal.js

```diff
@@ -17,6 +17,33 @@ Blockchain.createBlockchain("full-node", ()=>{}, async ()=>{
 
     Node.NodeServer.startServer();
 
+    let jsonRpcConfig = {
+        serverConfig: {
+            host: process.env.JSON_RPC_HOST,
+            port: process.env.JSON_RPC_PORT,
+        },
+
+        // @see express-basic-auth package for configuration (except isEnabled)
+        basicAuth: {
+            users: {},
+            isEnabled: false
+        },
+
+        // @see express-rate-limit package for configuration (except isEnabled)
+        rateLimit: {
+            windowMs: 60 * 1000, // 1 minute
+            max     : 60,        // limit each IP to 100 requests per windowMs,
+            isEnabled: true
+        }
+    };
+
+    if (typeof process.env.JSON_RPC_USERNAME !== "undefined" && typeof process.env.JSON_RPC_PASSWORD !== "undefined")
```

Comment:
If the username/password are not set, the authentication is disabled. Some methods that require authentication will not be available.